app/models/chargeback/consumption_without_rollups.rb

```diff
@@ -57,7 +57,7 @@ def current_value(metric, sub_metric = nil)
                              when 'derived_vm_numvcpus' # Allocated CPU count
                                resource.hardware.try(:cpu_total_cores)
                              when 'derived_memory_available'
-                               resource.hardware.try(:memory_mb)
+                               resource.hardware.try(:ram_size_in_bytes)
```

Comment:
think this is still memory_mb
and use `&.` since that is what try is doing here